eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -73,9 +78,25 @@ def lsqfitter(lc, model, meta, log, calling_function='lsq', **kwargs):
 
     # Get the best fit params
     fit_params = results.x
-
+    # Create table of results
+    t_results = table.Table([freenames, fit_params], 
+                            names=("Parameter", "Median")  )  
+
+    # Save transmission spectrum
+    # indices of fitted rps for all channels
+    ind_spec = np.array([i for i in range(len(freenames)) if 'rp' in freenames[i]])
+    if len(ind_spec):
+        # get wavelengths at middle of bin
+        wave_low = meta.wave_low[lc.fitted_channels]
+        wave_hi = meta.wave_hi[lc.fitted_channels]
+        wave_mid = (wave_hi+wave_low)/2.
+        t_spec = table.Table([wave_low, wave_mid, wave_hi, fit_params[ind_spec]**2.*1e6, \
```

Comment:
Instead of ppm, maybe we should list values in %? We should discuss pros and cons next week